tests/src/test/javascript/crypto/EccCryptoContextSuite.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 2016 Netflix, Inc.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * ECC crypto context unit tests.
+ *
+ */
+describe("EccCryptoContext", function() {
+    var publicKeyA;
+    var privateKeyA;
+    var publicKeyB;
+    var privateKeyB;
+
+    var ECDSA_KEYPAIR_A = {
+        publicKeyJSON: {
+            "kty": "EC",
+            "crv": "P-256",
+            "x":   "MKBCTNIcKUSDii11ySs3526iDZ8AiTo7Tu6KPAqv7D4",
+            "y":   "4Etl6SRW2YiLUrN5vfvVHuhp7x8PxltmWWlbbM4IFyM",
+            "use": "sig",
+            "kid": "A"
+         },
+        privateKeyJSON: {
+            "kty": "EC",
+            "crv": "P-256",
+            "x":   "MKBCTNIcKUSDii11ySs3526iDZ8AiTo7Tu6KPAqv7D4",
+            "y":   "4Etl6SRW2YiLUrN5vfvVHuhp7x8PxltmWWlbbM4IFyM",
+            "d":   "870MB6gfuTJ4HtUnUvYMyJpr5eUZNP4Bk43bVdj3eAE",
+            "use": "sig",
+            "kid": "Apriv"
+         }
+     };
+
+    var ECDSA_KEYPAIR_B = {
+        publicKeyJSON: {
+            "kty": "EC",
+            "crv": "P-256",
+            "x":   "10jF0O9oX8GOXxxHgC9gktZbduHn0K3XI5EDum_fOkc",
+            "y":   "eBVHsIhtASlzUIzGnMAl0TDfj0pqgldZrbsZobEL-78",
+            "use": "sig",
+            "kid": "B"
+         },
+        privateKeyJSON: {
+            "kty": "EC",
+            "crv": "P-256",
+            "x":   "10jF0O9oX8GOXxxHgC9gktZbduHn0K3XI5EDum_fOkc",
+            "y":   "eBVHsIhtASlzUIzGnMAl0TDfj0pqgldZrbsZobEL-78",
+            "d":   "TBP2kKufnhTHEf88VPcOIPDlk8uAFlLgi7C1iv86huY",
+            "use": "sig",
+            "kid": "B"
+         }
+     };
+
+    /** Random. */
+    var random = new Random();
+    /** Message. */
+    var message = new Uint8Array(32);
+    random.nextBytes(message);
+    /** MSL context. */
+    var ctx = null;
+    var initialized = false;
+
+    beforeEach(function() {
+        var extractable = true;
+        var _algo = { name: "ECDSA", namedCurve: "P-256" };
+
+        if (!initialized) {
+            runs(function() {
+                mslCrypto["importKey"]("jwk", ECDSA_KEYPAIR_A.publicKeyJSON, _algo, extractable, ["verify"])
+                    .then(function (key) {
+                        publicKeyA = key;
+                    },
+                    function (err) {
+                        console.log(err);
```

Comment:
To make Jasmine fail you need to trigger Jasmine. See the other JS unit tests when errors occur.